styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Inter', sans-serif;
+}
+.cta {
+    background: #2EC5CE;
+    display: flex;
+    justify-content: space-around;
+    margin: auto;
+
+}
+.cta-left {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    margin: 70px 75px 70px 165px;
+    gap: 48px;
+    
+}
+.cta-left-top {
+    width: 540px;
+    
+}
+.cta-text {
+    gap: 8px;
+    color: #18191F;
+    display: flex;
+    flex-direction: column;
+}
+.cta-text-heading {
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 64px;
+    height: 130px;
+}
+.cta-text-content {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 32px;
+    
+}
+.cta-left-bottom {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+}
+.cta-bottom-image {
+    display: flex;
+    gap: 12px;
+    cursor: pointer;
+}
+.cta-bottom-text {
+    font-size: 18px;
+    line-height: 28px;
+}
+.cta-right {
+    display: flex;
+    gap: 48px;
+    margin-right: 165px;
+}
+.cta-second-image {
```

Comment:
I think this class selector style is not required. check through inspecting and remove if it's not affecting